src/odemis/util/polar.py

```diff
@@ -78,6 +78,7 @@ def _extractAngleInformation(data, hole, dtype=None):
 
     # Crop the input image to half circle (set values outside of half circle zero)
 
+    # TODO cropped image wrong
```

Comment:
Is that indeed a problem? Can you describe it a little bit more? Or remove the comment.